resources/rest-service/cloudify/install_agent.py

```diff
@@ -159,6 +173,13 @@ def install(self):
         finally:
             self.runner.rm_dir(path)
 
+    def _get_certificate_file(self):
+        if not self.cloudify_agent.get('verify_rest_certificate'):
+            return None
+        with tempfile.NamedTemporaryFile(delete=False) as f:
+            f.write(self.cloudify_agent['rest_cert_content'])
```

Comment:
perhaps create the file under the temp dir `path` that gets created and deleted already.
currently this file is created but never gets deleted.